src/main/java/appeng/util/Platform.java

```diff
@@ -803,7 +805,20 @@ public static boolean isWrench( final EntityPlayer player, final ItemStack eq, f
 			{ // explodes without BC
 
 			}
-
+			
+			// CoFH Wrench Support
+            try {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)